src/Components/Backoffice/BackofficeHeader.js

```diff
@@ -0,0 +1,92 @@
+import React, {useState} from 'react';
+import {styled,useTheme} from '@mui/material/styles';
+import MenuIcon from '@mui/icons-material/Menu';
+import ChevronRight from '@mui/icons-material/ChevronRight';
+import ChevronLeft from '@mui/icons-material/ChevronLeft';
+import {IconButton,
+        Box,
+        Typography,
+        Drawer,
+} from '@mui/material';
+import MuiAppBar from '@mui/material/AppBar';
+
+const BackofficeHeader = () => {
+
+  const drawerWidth = 240;
+
+  const DrawerHeader = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    padding: theme.spacing(0, 1),
+    ...theme.mixins.toolbar,
+    justifyContent: 'flex-end',
+  }));
+  const AppBar = styled(MuiAppBar, {
+    shouldForwardProp: (prop) => prop !== 'open',
+    })(({ theme, open }) => ({
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.leavingScreen,
+    }),
+    ...(open && {
+      width: `calc(100% - ${drawerWidth}px)`,
+      marginLeft: `${drawerWidth}px`,
+      transition: theme.transitions.create(['margin', 'width'], {
+        easing: theme.transitions.easing.easeOut,
+        duration: theme.transitions.duration.enteringScreen,
+      }),
+    }),
+  }));
+  const theme = useTheme();
+
+  const [open,setOpen] = useState(false)
+
+  const handleDrawerOpen = () => {
+    setOpen(true);
+  };
+
+  const handleDrawerClose = () => {
+    setOpen(false);
+  };
+
+
+  return (
+    <>
+      <AppBar position="static" sx={{heigth:20,display:'flex',flexDirection:'row',alignItems:'center'}}>
+        <IconButton
+          color="inherit"
+          aria-label="open drawer"
+          onClick={handleDrawerOpen}
+          edge="start"
+          sx={{ ml: 2,mr:2}}
+        >
+          <MenuIcon/>
+        </IconButton>
+        <Typography variant="h6" component="div">
+          Somos Mas
+        </Typography>
+      </AppBar>
+      <Drawer
+        sx={{
+          width: drawerWidth,
+          flexShrink: 0,
+          '& .MuiDrawer-paper': {
+          width: drawerWidth,
+          boxSizing: 'border-box',
+        },
```

Comment:
move this styles into a style file

Reply:
I think its the better way for styling MUI components, at least its how it appear on the documentation. For better css code we need to create a theme that affects all the application.

Reply:
you can continue having this like it is and moving it to other file.